### ISSUE TITLE
spirv-fuzz: handle case where maximum id bound is reached

### ISSUE THREAD

**Issue by afd** [KhronosGroup/SPIRV-Tools#2541]
spirv-fuzz will add ids to a module, and could hit the maximum id bound.

Some care will ultimately be required to (a) pack ids, both in transformations and in the module, to avoid reaching the bound, and (b) bail out gracefully in the case where the id bound is truly reached.

**Comment by afd** [comment]
@Vasniktel from one of our recent discussions I think we do this - there's a fuzzer return code for reaching the id bound, right?

**Comment by Vasniktel** [comment]
Yes, I think so. We can remove https://github.com/KhronosGroup/SPIRV-Tools/blob/4a59fd47634f51dce424364f81c05d1700f98121/source/fuzz/fuzzer_util.cpp#L102-L103
when we close this.
